src/test/kotlin/tornadofx/testapps/GotoViewApp.kt

```diff
@@ -10,11 +10,13 @@ class ContainerView : View("ContainerView") {
 
     override val root = borderpane {
         top {
-            button("Switch view").setOnAction {
-                if (center.lookup("#view1") != null)
-                    subView1.replaceWith(subView2, ViewTransition.Slide(0.2.seconds))
-                else
-                    subView2.replaceWith(subView1, ViewTransition.Slide(0.2.seconds, ViewTransition.Direction.RIGHT))
+            button("Switch view") {
```

Comment:
I usually do the same. If there is just a single statement inside a builder, I append it instead of adding a code block. Then I refactor it if I need more statements later.